server/src/main/java/org/elasticsearch/threadpool/ThreadPool.java

```diff
@@ -69,7 +69,7 @@
         public static final String GET = "get";
         public static final String ANALYZE = "analyze";
         public static final String INDEX = "index";
```

Comment:
silly question (only looking at the diff I don't have full context), do we need to keep this constant?

Reply:
@jpountz Yes; this is in 6.x where the `index` thread pool remains.